src/components/screens/home-screen/home-screen.tsx

```diff
@@ -26,6 +26,9 @@ interface Props extends WithStyles<typeof styles> {
  */
 interface State {
   redirectTo?: string;
+  moriningHour: Number,
+  noonHour: Number,
+  eveningHour: Number
```

Comment:
```suggestion
  moriningHour: number,
  noonHour: number,
  eveningHour: number
```
The correct type is "number"